docs/5min-overview.md

```diff
@@ -110,98 +109,15 @@ And if you’d rather join a pool, you will find list of available pools in the
-- 03.2023 - Improved difficulty adjustment algorithm (DAA)
-
-
-**Ecosystem**
-
-- 08.11.2021 - 3rd party cloud mining service at Mainnet Launch
-- 11.2021 - Joined the UTXO alliance & Bitcoin Association Switzerland
-- Q4.2021 - Mining ecosystem development:community pools, miners development, reference mining pool, and pool integration.
-- 12.01.2022 - First exchange listing: Gate.io
-- Q3.2022 - Initiation of marketing initiatives ( sponsoring of ErgoHack, AMAs, community competitions, campaigns, Tech Talk Series, etc)
-- Q3.2022 - 3rd party dApps development started on Alephium
-- 10.2022 - Dappnode Integration 
-- 11.2022 - Flux Integration
-- 11.2022 - Launch of Alephium Swag Shop
-- H2.2022 - Joint communication and marketing initiatives with UTXO Alliance members
-- 12.2022 - Reached over 265 community contributions 
-
-
-### Roadmap
-
-The network is young and growing rapidly. We will update the roadmap to reflect the latest priorities.
-
-**Core Platform**
-
-- [Feature] Desktop Wallet v2.0
-- [Launch] Mobile wallet (Android and iOS)
-- [Launch] Alephium’s Bridge Launch on Mainnet
-- [Feature & UX] Support streaming for blockchain events (tx, block, contract events)
-- [Feature & Security] Typescript SDK for off-chain transaction encoding/decoding
-- [Feature] P2P transaction endpoints and SDKs
-- [Community] Introduce AIP for Alephium Improvement Proposals
-- [Improvement] Improved web3 SDK with better DevX and more functionality
-- [Feature] Multisig support in SDK and wallets
-- [Security] Standard for transaction information display on wallet
-- [Improvement] Improved full node’s database system for better performance
-- [DevX & UX] Better error system for full node and its endpoints
-- [Feature] Design and implement fast sync for full node
-- [Feature] Smart contracts support in the Explorer
-- [Improvement] Enhanced UX and additional features for the Browser Wallet Extension
-- [Improvement] Improved NFT standard and prototype
-- [Launch] Additional dApps prototypes
-- [DevX] Better development experience and additional functionalities for the Ralph language
-- [Improvement] Continuous improvement of the documentation 
-- [Feature] Design and implement light node
-
-
-**Ecosystem**
-
-- Additional CEX listings
-- DEX listing 
-- Launch of 3rd party dApps (NFT, DEX, stablecoin, Alephium Name Services, oracle, etc) on Alephium
-- Ledger wallet integration
-- Additional bridges to other ecosystems
-- Introduction videos & articles for building dApps on Alephium
-- Integration with 3rd party wallets
-- First Alephium Hackathon
-- Grant program & Development Bounties V2
-- Improve developers' onboarding experience
-- Launch of the Ambassador program
-- Building-up community engagement
-- Additional marketing, partnership and knowledge-sharing initiatives with like-minded communities and projects.
-- Website revamp
+[Completed Milestones](https://alephium.org/#milestones)
 
+[Roadmap](https://alephium.org/#next)
```

Comment:
This link doesn't work (it doesn't scroll anywhere)

Reply:
It works for me somehow, I just tried again.